Small improvements to  tags and releases chapter (#204)
lnnrtwttkhn/version-control-book: chapters/tags-and-releases.qmd

On the Zenodo [start page](https://zenodo.org), you can click on "**Sign Up**" in the top right corner.
Here you either can directly use your **GitHub account** to sign up, or sign up using **your email** (or other accounts) and link your GitHub account later on.
To **link your GitHub account**, you can click on the arrow, next to your email, on the start page and select "GitHub".

> If you create an account on the normal [Zenodo](https://zenodo.org) site, you will have to create a new one to use the [Zenodo Sandbox](https://sandbox.zenodo.org), since they do not transfer.

Reviewer: ```suggestion
If you create an account on the [regular Zenodo site](https://zenodo.org), you will have to create a new account to use the [Zenodo Sandbox](https://sandbox.zenodo.org), since the accounts do not synchronize.
```

To upload a new release, visit the [Zenodo GitHub settings](https://sandbox.zenodo.org/account/settings/github/).
Here you should see all your uploaded repositories, with the option to sync them by selecting the "on" button.
If you now create a release, as discussed earlier, it will show up and be associated with a DOI.

> Please note that if you have created a release before syncing your repository, the release will not show up.

Reviewer: ```suggestion
Please note that if you have created a release *before* syncing your repository, the release will not show up.
```

> You can delete the GitHub release without deleting your tag and create a new release, which is going to show up.

Reviewer: ```suggestion
You can delete the GitHub release without deleting your tag and create a new release, which is the going to appear on Zenodo. 
```

For a pictured guide on how to upload a GitHub repository to Zenodo, you check out [this guide](https://coderefinery.github.io/github-without-command-line/doi/) from the [Code Refinery](https://coderefinery.org).

> #### DOI badge

Reviewer: ```suggestion
#### Markdown DOI badge
```

After you have created a release in your synced repo, a DOI badge should show up in your [Zenodo GitHub settings](https://sandbox.zenodo.org/account/settings/github/) next to the repository name.

> When you click on it there should be a section called **Markdown** with Markdown syntax below.

Reviewer: ```suggestion
When you click on it there should be a section called **Markdown** with Markdown syntax below, for example:
```

> E.g:

Reviewer: ```suggestion
```

```
[![DOI](https://sandbox.zenodo.org/badge/731953735.svg)](https://sandbox.zenodo.org/doi/10.5072/zenodo.71301)

```

> You can copy paste this syntax into your `README.md` on GitHub to create a clickable DOI badge

Reviewer: ```suggestion
You can copy and paste this syntax into your `README.md` file on GitHub to create a clickable DOI badge.
```
